Nodejs/Product/Nodejs/Options/NodejsNpmOptionsControl.Designer.cs

```diff
@@ -160,14 +166,61 @@ partial class NodejsNpmOptionsControl {
+            this.tableLayoutPanel2.RowStyles.Add(new System.Windows.Forms.RowStyle(System.Windows.Forms.SizeType.Absolute, 20F));
+            this.tableLayoutPanel2.Size = new System.Drawing.Size(388, 148);
             this.tableLayoutPanel2.TabIndex = 4;
             // 
+            // groupBox3
+            // 
+            this.groupBox3.AutoSize = true;
+            this.groupBox3.AutoSizeMode = System.Windows.Forms.AutoSizeMode.GrowAndShrink;
+            this.groupBox3.Controls.Add(this.tableLayoutPanel5);
+            this.groupBox3.Dock = System.Windows.Forms.DockStyle.Top;
+            this.groupBox3.Location = new System.Drawing.Point(6, 131);
+            this.groupBox3.Margin = new System.Windows.Forms.Padding(6, 3, 6, 3);
+            this.groupBox3.Name = "groupBox3";
+            this.groupBox3.Size = new System.Drawing.Size(376, 14);
+            this.groupBox3.TabIndex = 4;
+            this.groupBox3.TabStop = false;
+            this.groupBox3.Text = "Type Acquisition";
+            // 
+            // tableLayoutPanel5
+            // 
+            this.tableLayoutPanel5.AutoSize = true;
+            this.tableLayoutPanel5.AutoSizeMode = System.Windows.Forms.AutoSizeMode.GrowAndShrink;
+            this.tableLayoutPanel5.ColumnCount = 1;
+            this.tableLayoutPanel5.ColumnStyles.Add(new System.Windows.Forms.ColumnStyle(System.Windows.Forms.SizeType.Percent, 50F));
+            this.tableLayoutPanel5.Controls.Add(this._enableAutoTypingsAcquisition, 0, 0);
+            this.tableLayoutPanel5.Dock = System.Windows.Forms.DockStyle.Top;
+            this.tableLayoutPanel5.Location = new System.Drawing.Point(3, 16);
+            this.tableLayoutPanel5.Name = "tableLayoutPanel5";
+            this.tableLayoutPanel5.RowCount = 1;
+            this.tableLayoutPanel5.RowStyles.Add(new System.Windows.Forms.RowStyle(System.Windows.Forms.SizeType.Percent, 50F));
+            this.tableLayoutPanel5.Size = new System.Drawing.Size(370, 23);
+            this.tableLayoutPanel5.TabIndex = 1;
+            // 
+            // _enableAutoTypingsAcquisition
+            // 
+            this._enableAutoTypingsAcquisition.AutoSize = true;
+            this._enableAutoTypingsAcquisition.Checked = true;
+            this._enableAutoTypingsAcquisition.CheckState = System.Windows.Forms.CheckState.Checked;
+            this._enableAutoTypingsAcquisition.Dock = System.Windows.Forms.DockStyle.Top;
+            this._enableAutoTypingsAcquisition.Location = new System.Drawing.Point(3, 3);
+            this._enableAutoTypingsAcquisition.Name = "_enableAutoTypingsAcquisition";
+            this._enableAutoTypingsAcquisition.Size = new System.Drawing.Size(364, 17);
+            this._enableAutoTypingsAcquisition.TabIndex = 2;
+            this._enableAutoTypingsAcquisition.Text = "Enable automatic typings acquisition for JavaScript projects";
```

Comment:
-> "Automatically download IntelliSense typings for JavaScript projects (ES6 Preview)"